spec/example_app/app/models/blog.rb

```diff
@@ -0,0 +1,5 @@
+module Blog
+  def self.table_name_prefix
+    "blog_"
```

Comment:
This is produced by the `rails generate model Blog::Post` command. Yes, I expect that most of the users will be do the same.